lisa/tools/ntpstat.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any, cast
+
+from retry import retry
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+
+from .gcc import Gcc
+from .git import Git
+
+
+class Ntpstat(Tool):
+    repo = "https://github.com/darkhelmet/ntpstat"
+    __not_sync = "unsynchronised"
+
+    @property
+    def command(self) -> str:
+        return self._command
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        self._command = "ntpstat"
+
+    def _install_from_src(self) -> None:
+        posix_os: Posix = cast(Posix, self.node.os)
+        posix_os.install_packages([Git, Gcc])
```

Comment:
It doesn't need to install them here, the `tools[Git]` will trigger the installation.